pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -31,6 +31,7 @@ let
   rlinkLibs = if stdenv.isDarwin then [
     darwin.libobjc
     darwin.apple_sdk_11_0.frameworks.AppKit
+    darwin.apple_sdk_11_0.frameworks.VisionKit
```

Comment:
This doesn't seem to exist.

Reply:
Shouldn't it be `Vision`?

https://developer.apple.com/documentation/vision/

Reply:
Yes, let me try to build this myself locally.